src/main/java/uk/gov/dhsc/htbhf/smartstub/model/DepositFundsResponse.java

```diff
@@ -0,0 +1,16 @@
+package uk.gov.dhsc.htbhf.smartstub.model;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+@AllArgsConstructor(onConstructor_ = {@JsonCreator})
+public class DepositFundsResponse {
```

Comment:
This class shouldn't be in here, the endpoint is only for Benefits.

Reply:
Removed.